src/api/greptime/v1/database.proto

```diff
@@ -56,3 +56,7 @@ message ObjectResult {
 message FlightDataRaw {
   repeated bytes raw_data = 1;
 }
+
+message FlightAppMeta {
+  uint32 affected_rows = 1;
+}
```

Comment:
Maybe it's better to put `affected_rows` as part of data? IMO metadata is something more generic like our `Header`. And in `Output` it is at the same level as `RecordBatch`. To my understanding, `RecordBatch` is for query request and `affected_rows` is for other DML (except `SELECT`) and DDL.